overrides.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2016 turbobytes
+//
+// This file is part of geoipdb, a library of GeoIP related helper functions
+// for TurboBytes stack.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package geoipdb
+
+import (
+	"errors"
+	"fmt"
+
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// asnOverride is what is stored in the overrides collection.
+type asnOverride struct {
+	asn  string
```

Comment:
You don't need bson.Setter if you express asn like so :-

    asn    string   `bson:"_id"`

no need to decorate the name field since it matches whats in db. However you might need to export `Asn`  and `Name`. Actually. it might be a good idea to export `asnOverride` so we could re-use this data structure from other places when we want to make changes.